eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -325,7 +325,7 @@ jobs:
 
     # Generate Documentation
     - pwsh: |
-        $setupFile="doxygen-1.8.18-setup.exe";
+        $setupFile="doxygen-1.8.20-setup.exe";
```

Comment:
@danieljurek, @weshaggard - can we make this available on `https://azuresdktooling.blob.core.windows.net`?

Reply:
I'm sure we can but I'm not even exactly sure who setup the azuresdktooling blob storage. @danieljurek was that you?

Reply:
It was me. This is to avoid having to download doxygen over http or from sourceforge.

Reply:
I was trying to download the 1.8.20 version from SourceForge just now and all of the mirrors I could find all fail. Timing is great for illustrating that this was the kind of problem I was trying to prevent using our blob storage.